libqtile/utils.py

```diff
@@ -209,3 +209,41 @@ def send_notification(title, message, urgent=False, timeout=10000):
         notifier.show()
     except Exception as exception:
         logger.error(exception)
+
+
+def get_default_terminal():
+    """Try to guess terminal."""
+    test_terminals = [
+            ['x-terminal-emulator', '-?'],
+            ['xterm', '-help'],
+            ['urxvt', '-help'],
+            ['st', '-v'],
+            ['gnome-terminal', '-?'],
+            ['terminator', '-v'],
+            ['lxterminal', '-v'],
+            ['mate-terminal', '-h'],
+            ['roxterm', '-?'],
+            ['tilda', '--help'],
+            ['guake', '-h'],
+            ['eterm', '-h'],
+            ['sakura', '-v'],
+            ['konsole', '--help'],
+            ['hyper', '-v'],
+            ['alacritty', '-V'],
+            ]
```

Comment:
`xfce4-terminal` is quite a common terminal as well.

`]` should be dedented :stuck_out_tongue:

Reply:
[termite](https://github.com/thestinger/termite) and [kitty](https://github.com/kovidgoyal/kitty) are also [somewhat popular](https://wiki.archlinux.org/index.php/Dotfiles).

Reply:
we don't need to cover all terminals. The main idea is to give user some working terminal. If we will see that list should be expanded - then new PRs would be welcome.

Reply:
> we don't need to cover all terminals.

Why not? To me, everything that is a terminal emulator and is in a non-user contributed repository (i.e. no AUR) of a mainstream distribution could make it in the list. ``termite``, ``kitty``, but also ``yakuake`` are in community or extra repos in Arch. ``yakuake`` is even in Debian stable (Quake-style emulator for KDE, like ``guake`` for Gnome).

This criterion is not satisfied by all programs in this list (e.g. ``roxterm`` (AUR: 10 votes, pop 0.03), ``hyper`` (AUR: 58 votes, pop 1.30), ``eterm`` (AUR: 25 votes, pop 0)) and that's in principle fine, but I see no reason to not extend the search list right now. There will be no better metric in the future to decide if a program is worth adding (telemetrics is not a suitable solution).

Reply:
I loved yakuake ~15 years ago. Ok, added kitty, termite, yakuake